### PR TITLE
Add ability to deploy to production

### PR DESCRIPTION
**Pull request by raws**
🤫

### CODE REVIEW

File: scripts/deploy.coffee [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+]
+
+emojis = [
+  ':+1:',
+  ':dance:',
+  ':shush:'
+]
+
+module.exports = (robot) ->
+  robot.respond /deploy\s+to\s+prod(?:uction)?/i, (msg) ->
```
Comment (xxv) [inline scripts/deploy.coffee]: add word boundary `\b` to first and last words?

File: scripts/deploy.coffee [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  'levelup-order-ahead-production',
+  'levelup-production',
+  'order-ahead-network-production',
+  'plutus-production'
+]
+
+emojis = [
+  ':+1:',
+  ':dance:',
+  ':shush:'
```
Comment (xxv) [inline scripts/deploy.coffee]: maybe also add `:spinner:`?

File: scripts/deploy.coffee (hunk 1 above)
Comment (raws) [reply scripts/deploy.coffee]: Ah, good thought, but the pattern is effectively bounded by the bot's name: https://github.com/hubotio/hubot/blob/a5178f5560251edcace24ca73956dbfcb9b3307f/src/robot.coffee#L109

So it'll only ever respond to "ralph deploy…", not "ralph plzdeploy".

File: scripts/deploy.coffee (hunk 2 above)
Comment (raws) [reply scripts/deploy.coffee]: Good idea. Addressed in 9a97705!

File: scripts/deploy.coffee [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+apps = [
+  'argos',
+  'levelup-order-ahead-production',
+  'levelup-production',
+  'order-ahead-network-production',
+  'plutus-production'
+]
+
+emojis = [
+  ':+1:',
```
Comment (JLoppert) [inline scripts/deploy.coffee]: Consider `:party_poop:`, `:partydumpster:`, and `:party_try_not_to_cry:` as valuable emotes

**Pull request merged**